CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Changes since v2.15
 ### Fixes
 
 ### Additions
+- The browser tests will now fail if there are any accessibility violations. (#2463)
```

Comment:
not sure if this needs to be in the changelog since it's not user-facing at all, but I don't mind